### PR TITLE
DEP PHP Support in CMS5

### PR DESCRIPTION
**Pull request by sabina-talipova**
## Parent issue
- https://github.com/silverstripe/silverstripe-framework/issues/10402

### CODE REVIEW

**Comment by sabina-talipova** [comment]
Requires 
- https://github.com/silverstripe/silverstripe-userforms/pull/1182

Passed tests in local environment.

File: _config.php [hunk 1]
```diff
@@ -1,14 +0,0 @@
-<?php
```
Comment (maxime-rainville) [inline _config.php:1]: Not having a `_config.php` file at all might cause problems. From memory there's a bit in framework that won't recognise a packages is a silverstripe module if that file doesn't exist?

Just leave an empty `_config.php` file in place with:
```php
<?php

```

**Review by maxime-rainville — CHANGES_REQUESTED** [review]

File: _config.php (hunk 1 above)
Comment (sabina-talipova) [reply _config.php:1]: DONE. I restored file and left message.

Comment (maxime-rainville) [reply _config.php:1]: The problem wasn't removing the deprecation messages. The problem was that we need a `_config.php` file  in the root of the module ... even if it's completely empty.

Comment (maxime-rainville) [reply _config.php:1]: e.g.: https://github.com/silverstripe/silverstripe-blog/blob/3/_config.php

Comment (sabina-talipova) [reply _config.php:1]: DONE

**Review by maxime-rainville — APPROVED** [review]
LGTM

**Pull request merged**